GoogleChromeLabs/ps-analysis-tool
# Add a favicon to the downloadable report and CLI dashboard.

## Description

Add a favicon to the downloadable report and CLI dashboard.

## Relevant Technical Choices

Add favicon link to publish html files header.

## Testing Instructions

- Download reports from CLI and extension to check if it has a favicon.
- Check if CLI dashboard has favicon

## Additional Information:



## Screenshot/Screencast



---

## Checklist






Fixes #741
